libbeat/dashboards/dashboards.go

```diff
@@ -106,12 +104,7 @@ func ImportDashboards(
 
 		esLoader.statusMsg("Elasticsearch URL %v", esLoader.client.Connection.URL)
 
-		majorVersion, _, err := getMajorAndMinorVersion(esLoader.version)
-		if err != nil {
-			return fmt.Errorf("wrong Elasticsearch version: %v", err)
-		}
-
-		if majorVersion < 6 {
+		if esLoader.version.Major < 6 {
```

Comment:
Not related to this PR but I wonder if we should remove support for this.

@jsoriano More related to your other PR with removing 5.x dashboards.

Reply:
We can consider to keep this in libbeat in case some community beat still wants to try to keep support for 5.x?

If not, I can remove it in in #8927, along with the `importViaES` method.

Reply:
I would rather remove it to reduce complexity in the code. Can also be done in a follow up PR to keep things smaller.